src/nfa/operations.cc

```diff
@@ -516,6 +516,16 @@ bool mata::nfa::Nfa::is_prfx_in_lang(const Run& run) const {
 }
 
 bool mata::nfa::Nfa::is_lang_empty(Run* cex) const {
+    //TOOD: hot fix for performance reasons for TACAS.
+    // Perhaps make the get_useful_states return a witness on demand somehow.
+    if (!cex) {
+        BoolVector useful_states = get_useful_states();
+        for (auto useful: useful_states)
```

Comment:
Please, lets not use `auto` (even if it is hotfix).

Reply:
right
(although here, if we change BoolVector to something else, then it will stop working)

Reply:
I get your point, and this is not that bad usage of `auto`; still I think that explicit typing is more readable, especially in future.

Reply:
true true